corehq/apps/export/forms.py

```diff
@@ -344,7 +345,7 @@ def get_multimedia_task_kwargs(self, export, download_id):
             'app_id': export.app_id,
             'xmlns': export.xmlns if hasattr(export, 'xmlns') else '',
             'export_id': export.get_id,
-            'zip_name': 'multimedia-{}'.format(export.name),
+            'zip_name': 'multimedia-{}'.format(unidecode(export.name)),
```

Comment:
it probably would have worked - that's what I did initially, then I followed this var around to see where it's used (it broke further down the line from the same thing), and saw that we use `unidecode` elsewhere for export filenames, so I followed that for consistency.